### PR TITLE
(GH-38) Update for 0.28.0

### PR DESCRIPTION
**Pull request by glennsarti**
This commit updates the documentation for Extension version 0.28.0

### CODE REVIEW

File: vscode/content/en/docs/Extension Settings.md [hunk 1]
```diff
@@ -98,11 +98,11 @@ Default: `True`
 
 #### puppet.installDirectory
 
-The fully qualified path to the Puppet install directory. This can be a PDK or Puppet Agent installation. For example: 'C:\Program Files\Puppet Labs\Puppet' or '/opt/puppetlabs/puppet'. If this is not set the extension will attempt to detect the installation directory
```
Comment (jpogran) [inline vscode/content/en/docs/Extension Settings.md:101]: Did you mean to remove this?

Comment (glennsarti) [reply vscode/content/en/docs/Extension Settings.md:101]: That was automated so... yes?

Comment (jpogran) [reply vscode/content/en/docs/Extension Settings.md:101]: why tho? We still have these settings and we haven't changed the text for them?

Comment (glennsarti) [reply vscode/content/en/docs/Extension Settings.md:101]: Ahhh found it.  The text changed from `description` to `markdownDescription`

Comment (glennsarti) [reply vscode/content/en/docs/Extension Settings.md:101]: Updated and changed this to only update to 0.28.0.  The extension doesn't yet have changelog for 1.0.0

**Review by jpogran — APPROVED** [review]

**Pull request merged**